docs/content/docs/build/dds.md

```diff
@@ -27,7 +27,7 @@ However, a DDS is not *just* a local object. A DDS can also be changed by other
 
 {{% callout tip %}}
 
-Most distributed data structures are prefixed with "Shared" by convention. *SharedMap*, *SharedMatrix*, *SharedString*,
+Most distributed data structures are prefixed with "Shared" by convention. *SharedMap*, *SharedTree*, *SharedString*,
```

Comment:
Nit: I would probably remove `SharedString` from this list. We don't really want our SDK customers using SharedString going forward.